src/context/json.test.ts

```diff
@@ -36,5 +36,33 @@ describe('json', () => {
         '"2020-01-01T00:00:00.000Z"',
       )
     })
+
+    it('should allow merging with a prior body', () => {
+      const firstNonNestedObject = json({ firstName: 'John' }, { merge: true })
+      const secondNonNestedObject = json({ lastName: 'Santa' }, { merge: true })
+      const thirdNonNestedObject = json({ lastName: 'Doe' }, { merge: true })
```

Comment:
I think we can limit this to 2 bodies in total. The fact that given JSON gets merged deeply is asserted in the unit tests for the `mergeRight` utility, there's no need to verify that once more. In `json` utility test we just test that given `merge: true` it merges 2 bodies, if not—it doesn't.